commands/channel_test.go

```diff
@@ -669,10 +669,10 @@ func (s *MmctlUnitTestSuite) TestListChannelsCmd() {
 
 		err := listChannelsCmdF(s.client, cmd, args)
 
-		s.Require().Nil(err)
+		s.Require().ErrorContains(err, "unable to find team \""+teamID+"\"")
 		s.Len(printer.GetLines(), 0)
 		s.Len(printer.GetErrorLines(), 1)
-		s.Require().Equal(printer.GetErrorLines()[0], "Unable to find team '"+teamID+"'")
+		s.Require().Equal(printer.GetErrorLines()[0], "unable to find team \""+teamID+"\"")
```

Comment:
I know this was already wrong in the original code but could you swap the two arguments? `Equal` expects the expected message as the first string and the returned one as the second. Otherwise the error messages are the wrong way around. This is only limited to `Equal`, though. `EqualError` is the other way around.